EmpowerPlant/EmpowerPlantViewController.swift

```diff
@@ -8,21 +8,149 @@
 import UIKit
 import Sentry
 
-class EmpowerPlantViewController: UIViewController {
+class EmpowerPlantViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
 
+    let context = (UIApplication.shared.delegate as! AppDelegate).persistentContainer.viewContext
+    
+    let tableView: UITableView = {
+        let table = UITableView()
+        table.register(UITableViewCell.self, forCellReuseIdentifier: "cell")
+        return table
+    }()
+    
+    var products = [Product]()
+    
     override func viewDidLoad() {
         super.viewDidLoad()
         title = "Empower Plant"
-        // Do any additional setup after loading the view.
+
+        self.view.addSubview(tableView)
+        tableView.delegate = self
+        tableView.dataSource = self
+        
+        // Dev Note - Comment this out and to see the green background and no data in the rows
+        tableView.frame = view.bounds
+        
         configureNavigationItems()
+
+        // TODO - clear Db on startup, and insert the objects from server into the db
```

Comment:
or, could do products in memory, and only use the db for a Shopping Cart.

Regardless, DB needed to be tested.